independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/buildtool/maven/scala/pom.tpl.qute.xml

```diff
@@ -52,7 +52,12 @@
                         <arg>-deprecation</arg>
                         <arg>-feature</arg>
                         <arg>-explaintypes</arg>
+                        {#if scala.version.startsWith("2.12")}
```

Comment:
You forgot the trailing dot I mentioned on Zulip :)

Reply:
Already fixed :)

Reply:
YOur bot is too slow for me